package.json

```diff
@@ -14,7 +14,7 @@
     "check": "yarn node ./tasks/check/check.mjs",
     "clean:prisma": "rimraf node_modules/.prisma/client && node node_modules/@prisma/client/scripts/postinstall.js",
     "e2e": "node ./tasks/run-e2e",
-    "lint": "RWJS_CWD=packages/create-redwood-app/template eslint --config .eslintrc.js packages",
+    "lint": "RWJS_CWD=packages/create-redwood-app/templates/ts eslint --config .eslintrc.js --ignore-pattern Routes.js packages",
```

Comment:
`yarn lint` complains about the `Routes.js` file in the JS template. It says "NotFoundPage" isn't defined, which is true, but it doesn't understand that Redwood projects take care of importing it. While we need a more straightforward linting setup, I don't think it should be done in this PR